hushline/templates/settings/profile.html

```diff
@@ -5,6 +5,20 @@
   aria-labelledby="profile-tab"
 >
   {# TODO: much of this was copy/pasted to "settings/alias.html" and will need to have updates synced there #}
+
+  {% if is_premium_enabled and (user.tier_id == 1 or user.tier_id == None) %}
```

Comment:
Nit. Move this message above the `TODO` so it's more obvious what the `TODO` references

Reply:
Fixed in 28932912dd6fbb8ede5c3ecbfc05660285da3064